interns/tamara-nicole-medrano/journals/index.html

```diff
@@ -13,6 +13,23 @@ <h1>Journals</h1>
     width="800px"
     height="500px"> 
   </object>
+
+ <h2>Journals</h2>
+ 
+  <object
+    data="./Tamara Medrano - Week 2 Journal Entries .pdf"
```

Comment:
Remove the extra space from your local file first and then update this line
```suggestion
    data="./Tamara Medrano - Week 2 Journal Entries.pdf"
```